client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/email-provider-stacked-features.tsx

```diff
@@ -50,4 +50,33 @@ const EmailProviderStackedFeatures: FunctionComponent< EmailProviderStackedFeatu
 	);
 };
 
-export default EmailProviderStackedFeatures;
+export interface EmailProviderStackedFeaturesProps {
+	features: TranslateResult[];
+}
+
+export const EmailProviderStackedFeaturesToggleButton: FunctionComponent< EmailProviderStackedFeaturesToggleButton > = (
+	props
+) => {
+	const translate = useTranslate();
+	const { handleClick, isRelatedContentExpanded } = props;
+
+	return (
+		// eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/interactive-supports-focus
+		<span
+			role="button"
+			className="email-provider-stacked-features__toggle-button"
+			onClick={ handleClick }
+		>
```

Comment:
Why can we not use a standard `Button` element? These accessibility lint checks should probably not be skipped.